catroid/src/main/java/org/catrobat/catroid/ui/fragment/ListActivityFragment.java

```diff
@@ -102,6 +102,7 @@ public void onDestroyActionMode(ActionMode mode) {
 				clearCheckedItems();
 			} else {
 				copyCheckedItems();
+				clearCheckedItems();
```

Comment:
this is called in both branches of the if.. it can be done once below the if/else, and by altering the condition, the else branch would be obsolete